app/controllers/activity_forms_controller.rb

```diff
@@ -151,12 +151,15 @@ def assign_default_collaboration_type_value_if_nil
   end
 
   def page_title(step)
-    if step == :linked_activity
-      @activity.is_oda ? t("page_title.activity_form.show.linked_non_oda_activity") : t("page_title.activity_form.show.linked_oda_activity")
-    elsif step == :identifier && @activity.programme?
+    if step == :identifier && @activity.programme?
       t("page_title.activity_form.show.identifier_level_b")
     else
-      t("page_title.activity_form.show.#{step}", sector_category: t("activity.sector_category.#{@activity.sector_category}"), level: t("page_content.activity.level.#{@activity.level}"))
+      t(
+        "page_title.activity_form.show.#{step}",
```

Comment:
Do you think we could add a bit more to the commit message here to say that these arguments will be ignored in most cases and only used where we interpolate the `linked_activity_type` (for example)? At a quick glance, it feels as though we actually care about their values, when we don't really.